skt/executable.py

```diff
@@ -719,6 +720,14 @@ def setup_parser():
         type=str,
         help="Additional options to pass to make"
     )
+    parser_build.add_argument(
+        "--make-target",
+        dest="make_target",
+        type=str,
+        default='targz-pkg',
+        choices=('targz-pkg', 'binrpm-pkg'),
```

Comment:
The commit says `Only targz-pkg is supported for now.` hehe

Reply:
Oops. :) That's coming in the next patch.